addons/core/XEH_preInit.sqf

```diff
@@ -16,28 +16,36 @@ GVAR(WeaponConfigCacheNamespace) = false call CBA_fnc_createNamespace;
 VARIABLE_DEFAULT(TFAR_Teamspeak_Channel_Name,"");
 VARIABLE_DEFAULT(TFAR_Teamspeak_Channel_Password,"");
 
+VARIABLE_DEFAULT(TFAR_DefaultRadio_Backpack_West,"TFAR_rt1523g");
+VARIABLE_DEFAULT(TFAR_DefaultRadio_Backpack_East,"TFAR_mr3000");
+VARIABLE_DEFAULT(TFAR_DefaultRadio_Backpack_Independent,"TFAR_anprc155");
 
-if (hasInterface) then {//Clientside Variables
```

Comment:
I am fine if you move this down to make variables availalbe on the server. But please don't reorder stuff for no reason.

Reply:
It's made on purpose. We have markers system based on radio parameters and some of variables needed on server. Double checked not to broke something.

Reply:
Yeah as I said. No problem. If you need them on the server just move the hasInterface check to after the variables.
But no reason to reorder them.

Reply:
There are some variables needed only on clients and I don't want to consume server memory for them.

Reply:
That's about 200 bytes at most.. Irrelevant.